### ISSUE TITLE
batch15 images seem to be corrupted

### ISSUE THREAD

**Issue by DmitryUlyanov** [lbasek/image-denoising-benchmark#11]
They are just black, is it ok?

**Comment by lbasek** [comment]
yep, that is unfortunately okay. When we created dataset, we are from original photographed image (RENOIR dataset) cropped small chunk and looks like in `batch15` we randomly pick black spot on the image.

**Comment by DmitryUlyanov** [comment]
But why the algorithms work so bad on batch15 image then? Did you test them on the full images, not on the chunks from this repo?

**Comment by lbasek** [comment]
Not sure, unfortunately I don't remember that I have issue. 
You can generate your own dataset with `python3 util/create_dataset.py`. (*but change the dir path before)
No, that was too demanding because that experiment would take a long time for full size images.
